person.rb

```diff
@@ -3,14 +3,15 @@
 class Person
   attr_accessor :name, :age
 
-  attr_reader :id
+  attr_reader :id, :rental
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     @id = Random.rand(1..1000)
     @age = age
     @name = name
     @parent_permission = parent_permission
     @corrector = Corrector.new
+    @rental = []
   end
```

Comment:
Thank you for your time and constructive review.

Reply:
I think both methods were still working because I did a number of tests on my code before submitting it for review. I have however, implemented your suggestions because it is easy to manage and is more optimized. Thanks.